packages/tasks/src/tasks/text-generation/about.md

```diff
@@ -1,10 +1,10 @@
-This task covers guides on both [text-generation](https://huggingface.co/models?pipeline_tag=text-generation&sort=downloads) and [text-to-text generation](https://huggingface.co/models?pipeline_tag=text2text-generation&sort=downloads) models. Popular large language models that are used for chats or following instructions are also covered in this task. You can find the list of selected open-source large language models [here](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard), ranked by their performance scores.
+This task covers guides on both [text-generation](https://huggingface.co/models?pipeline_tag=text-generation&sort=downloads) and [text-to-text generation](https://huggingface.co/models?pipeline_tag=text2text-generation&sort=downloads) models. Popular large language models that are used for chats or following instructions are also covered in this task. You can find the list of selected open-source large language models [here](https://huggingface.co/spaces/open-llm-leaderboard/open_llm_leaderboard), ranked by their performance scores.
 
 ## Use Cases
 
 ### Instruction Models
 
-A model trained for text generation can be later adapted to follow instructions. One of the most used open-source models for instruction is OpenAssistant, which you can try [at Hugging Chat](https://huggingface.co/chat).
+A model trained for text generation can be later adapted to follow instructions. You can try some of the most powerful instruction-tuned open access models like Mixtral 8x7B, Cohere Command R+, Meta Llama3 70B, etc [at Hugging Chat](https://huggingface.co/chat).
```

Comment:
I would suggest to not specify the models, as otherwise we'll have to maintain the list

Reply:
IMO, it's good to add the model names, helps with SEO I think. I'm happy to take ownership of this once every month.